src/main.rs

```diff
@@ -59,32 +62,32 @@ fn parse_matches() -> clap::ArgMatches {
             Arg::with_name("slots_per_epoch")
                 .long("slots-per-epoch")
                 .takes_value(true)
-                .help("override the number of slots in an epoch"),
+                .help("override the number of slots in an epoch. Default for cluster_type: development -> 8192.
```

Comment:
Or crap, I didn't realize it was still 8192 for dev clusters. Might be worth revisiting in separate work